lib/galaxy/queue_worker.py

```diff
@@ -63,6 +63,7 @@ def _get_new_toolbox(app):
     and then adding pre-existing data managers from the old toolbox to the new toolbox.
     """
     from galaxy import tools
+    from galaxy.tools.special_tools import load_lib_tools
     from galaxy.tools.toolbox.lineages.tool_shed import ToolVersionCache
```

Comment:
And then the tests all fail. It'd be nice to unwind all of that - but we should probably do it in its own PR.